pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3653,9 +3654,112 @@ func Test_server_MultiInsert(t *testing.T) {
 		beforeFunc func(args)
 		afterFunc  func(args)
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
Should we implement this code in the `t.Run` (line. 4311~ )?

Reply:
It is used in `args` field in use case and the context is shared across all of the requests in test case, I think it is ok to implement it here, what do you think?

Reply:
Thank you for your comment. I understand.